src/Common/Admin/Entities/Link.php

```diff
@@ -69,7 +83,7 @@ public function render() {
 			esc_url( $this->url ),
 			esc_attr( $this->get_classes() ),
 			$this->get_attributes(), // phpcs:ignore StellarWP.XSS.EscapeOutput,WordPress.Security.EscapeOutput
-			esc_html( $this->text )
+			$this->content ? (string) $this->content : esc_html( $this->text ) // phpcs:ignore StellarWP.XSS.EscapeOutput,WordPress.Security.EscapeOutput
```

Comment:
As part of the above-mentioned proposed changes, I would say that the entirety of the `render()` method be replaced with this:

```php
		?>
		<a href="<?php echo esc_url( $this->url ); ?>"
			class="<?php echo esc_attr( $this->get_classes() ); ?>"
			<?php echo $this->get_attributes(); // phpcs:ignore StellarWP.XSS.EscapeOutput,WordPress.Security.EscapeOutput ?>>
			<?php $this->content->render(); ?>
		<?php
```